src/PositionManager.sol

```diff
@@ -259,37 +236,56 @@ contract PositionManager is
     /// @dev Calling increase with 0 liquidity will credit the caller with any underlying fees of the position
     function _increase(
         uint256 tokenId,
-        PositionConfig calldata config,
         uint256 liquidity,
         uint128 amount0Max,
         uint128 amount1Max,
         bytes calldata hookData
-    ) internal onlyIfApproved(msgSender(), tokenId) onlyValidConfig(tokenId, config) {
+    ) internal onlyIfApproved(msgSender(), tokenId) {
+        (PositionInfo memory info, PoolKey memory poolKey) = getPoolPositionInfo(tokenId);
```

Comment:
are there gas savings if we dont load the entirety of PositionInfo? we're not using the truncated poolId obviously, and the ticks are just repacked to IPoolManager.ModifyLiquidityParams

something like this could be quite cool
```solidity
(
    PoolKey memory key,
    IPoolManager.ModifyLiquidityParams memory liquidityParams,
    bool hasSubscriber
) = veryAwesomeFunction(tokenId);

liquidityParams.liquidityDelta = liquidityChange;
```

theres also ways to pack it further since PoolKey is 528 bits, so we can fit tL/tU/hasSubscriber in the remaining bits of the 3rd word

Reply:
yeah was thinking the same thing.. I think I'm actually happy with it now that I have a version where the PositionInfo is packed (uint256).. since its the minimum we can basically return